config/routes.rb

```diff
@@ -37,6 +37,8 @@ def show_step(name)
     end
   end
 
+  get 'crime_applications/:id/delete', to: 'crime_applications#destroy', as: :confirm_application_deletion
```

Comment:
I think we can add this to the existing resources, it feels more clean and self-contained that way, like this:

```ruby
  resources :crime_applications, except: [:new, :update] do
    get :confirm_destroy, on: :member, action: :destroy
  end
```

On a related note, I've been thinking, perhaps is best to split the confirm and the actual destroy into two separate actions, to not overload the `#destroy` one? I know I mention doing this way but now I'm not so sure 😄 

It would be something like this:

```ruby
  resources :crime_applications, except: [:new, :update] do
    get :confirm_destroy, on: :member
  end
```

Note the removal of `action: :destroy`. That way we have 2 separate actions in the controller, the usual `destroy` and this new `confirm_destroy` and we don't have to check the request verb or anything like that.

Let me know what you think or we can jump into a call.